src/map/clif_packetdb.hpp

```diff
@@ -2409,6 +2409,11 @@
 	parseable_packet( 0x0B1C, sizeof( struct PACKET_CZ_PING ), clif_parse_dull, 0 );
 #endif
 
+#if PACKETVER_MAIN_NUM >= 20190522 || PACKETVER_RE_NUM >= 20190522 || PACKETVER_ZERO_NUM >= 20190515
```

Comment:
+	parseable_packet( HEADER_CZ_CASTLE_TELEPORT_REQUEST, sizeof(PACKET_CZ_CASTLE_TELEPORT_REQUEST), clif_parse_guild_castle_teleport_request, 0); 
please change it to
+	parseable_packet( HEADER_CZ_REQ_MOVE_GUILD_AGIT, sizeof(PACKET_CZ_REQ_MOVE_GUILD_AGIT), clif_parse_guild_castle_teleport_request, 0);